### ISSUE TITLE
Useless test in test/duration/info.test.js

### ISSUE THREAD

**Issue by tonysamperi** [moment/luxon#1046]
**Describe the bug**
The test `Duration#toObject accepts a flag to return config` under `test/duration/info.test.js` has 2 issues:

1. It's badly written as 2 arguments are passed to the `toEqual` method
2. It's pointless, because options for `Duration.toObject` were removed

It should be removed since toObject doesn't accepts the flag anymore (actually it doesn't accept anything).

**To Reproduce**

![image](https://user-images.githubusercontent.com/5957244/135910310-d608934c-4b22-4279-92e5-225d9797b2e6.png)

**Actual vs Expected behavior**
Actual and expected are the same, but it may be misleading.

**Desktop (please complete the following information):**

- OS: [e.g. iOS] Win 10
- Browser [e.g. Chrome 84, safari 14.0] Chrome and Node 13.13.0 (Tests)
- Luxon version [e.g. 1.25.0] 2.0.0
- Your timezone [e.g. "America/New_York"] Irrelevant, but "Europe/Rome"

**Additional context**
As you can see in the snippet below, the second argument passed in `toEqual` shouldn't be there.
```
test("Duration#toObject accepts a flag to return config", () => {
  expect(dur.toObject({ includeConfig: true })).toEqual(
    {
      years: 1,
      months: 2,
      days: 3,
    },
    {
      locale: "fr",
      numberingSystem: "beng",
      conversionAccuracy: "longterm",
    }
  );
});
```

**Comment by icambron** [comment]
agreed

**Comment by socraticDevBlog** [comment]
hello! I've removed the useless test 
https://github.com/moment/luxon/pull/1098

**Comment by icambron** [comment]
fixed in #1100
